frontend/mulighetsrommet-veileder-flate/src/views/tiltakstype-detaljer/ViewTiltakstypeDetaljer.tsx

```diff
@@ -15,7 +15,7 @@ interface RouteParams {
 }
 
 const ViewTiltakstypeDetaljer = () => {
-  const params = useParams<RouteParams>();
+  const params = useParams<'tiltakskode'>() as RouteParams;
```

Comment:
Denne endringen her skjønner jeg ikke helt? Er dette noe breaking siden vi går fra v5 til v6?

Reply:
Ja, det er nå kun strings som tas i mot som type-argument for `useParams`. Om man feks. har flere parameter så kan man gjøre slik `useParams<'tiltakskode | netflixKode | pinkode'>()`

Kan lese litt her fra Ryan Florence om hva han har å si om saken https://github.com/remix-run/react-router/issues/8200#issuecomment-1023658322

Reply:
Okay skjønner. Kunne vi ikke heller da gjøre litt "penere" med noe sånt ish (hvis det går, har ikke testa)?
```
type RouteParams = {
  tiltakskode: Tiltakskode
}

const params = useParams<RouteParams>()
```

Får bare litt vondt i magen av hardkoda stringer som flyter rundt :stuck_out_tongue_closed_eyes:

Reply:
Det fungerer dessverre ikke. Kan evt. lage en custom hook som feks. heter `useTiltakskodeSomParam` som ikke tar i mot noen argumenter.